asr1k_neutron_l3/models/netconf_yang/bgp.py

```diff
@@ -82,14 +82,6 @@ def __parameters__(cls):
         return [
             {'key': 'asn', 'id': True, 'yang-key': 'id'},
             {'key': 'vrf', 'yang-key': 'name'},
-            {'key': 'connected', 'yang-path': 'ipv4-unicast/redistribute', 'default': False,
-             'yang-type': YANG_TYPE.EMPTY},
-            {'key': 'static', 'yang-path': 'ipv4-unicast/redistribute', 'default': False,
-             'yang-type': YANG_TYPE.EMPTY},
-            {'key': 'connected', 'yang-path': 'ipv4-unicast/redistribute-vrf', 'default': False,
-             'yang-type': YANG_TYPE.EMPTY},
-            {'key': 'static', 'yang-path': 'ipv4-unicast/redistribute-vrf', 'default': False,
-             'yang-type': YANG_TYPE.EMPTY},
```

Comment:
Just to clarify: why can we remove this code from the model? It's not clear from the commit message

Reply:
in the `to_dict()` method of the `AddressFamily` class we have a block using connected/static, which was only used on versions older than 17.3. With newer firmware we announce networks via bgp network statement directly and not via `redistrbute connected/static`. See comment `# redistribute connected/static is only used with 16.9`.

I planned to reintroduce this to circumvent some fw locking in #114 but will reintroduce it once we've decided to go with this route.